contrib/mesos/pkg/controllermanager/controllermanager.go

```diff
@@ -305,6 +306,17 @@ func clientForUserAgentOrDie(config client.Config, userAgent string) *client.Cli
 	return kubeClient
 }
 
+// clientsetForUserAgentOrDie is going to replace clientForUserAgentOrDie
+func clientsetForUserAgentOrDie(config client.Config, userAgent string) release_1_1.Interface {
```

Comment:
This is a copy of the same function in cmd/kube-controller-manager/app/controllermanager.go?
Reuse the same func rather than duplicating?